common/app/common/InlineStyles.scala

```diff
@@ -93,21 +94,48 @@ object InlineStyles {
   }
 
   /**
+    * Execute some code n times, or until no exception is thrown
+    * (If n is 0, code still gets executed once)
+    */
+  def retry[T](n: Int)(r: => T)(implicit onFail: (Throwable, Int) => Unit) = {
```

Comment:
Thanks @TBonnin - unfortunately that `dispatch` module is designed for asynchronous requests so it requires the use of Futures which in this case are unnecessary and misleading. Hopefully this `retry` method won't be around forever, since if this extra logging helps us diagnose and fix the underlying problem, it will be redundant.